docs/tutorials/create-transactions.md

```diff
@@ -0,0 +1,272 @@
+---
+id: create-transactions
+title: Create Transactions
+sidebar_label: Create Transactions
+---
+To construct & process transactions you will need our API JavaScript library: [`near-api-js`](https://docs.near.org/docs/roles/developer/examples/near-api-js/introduction).
+
+  - All transactions require the following:
+      - `signerId` _(account ID of the transaction originator)_
+      - `signerPublicKey`
+      - `receiverId` _(account ID of the transaction recipient)_
+      - `nonceForPublicKey` _(each time a key is used the nonce value should be incremented by 1)_
+      - `actions` _( [[ click here ]](/docs/concepts/transaction#action) for supported arguments)_
+      - `blockHash` _(a current block hash (within 12hrs) to prove the transaction was recently created)_
+
+  See [Transaction Class](https://near.github.io/near-api-js/classes/_transaction_.transaction.html) for a more in depth outline.
+___
+
+# Create a "token transfer" transaction
+
+## Setup
+1) Clone the [transaction-examples](https://github.com/near-examples/transaction-examples) repository by running:
+
+```bash
+git clone https://github.com/near-examples/transaction-examples.git
+```
+
+2) Follow [setup instructions](https://github.com/near-examples/transaction-examples/blob/master/readme.MD#prerequisites)
+___
+
+## Imports
+In the first two lines of code in [`send-tokens.js`](https://github.com/near-examples/transaction-examples/blob/master/send-tokens.js#L1-L2), we:
+  1) Importing the NEAR API JavaScript library
+  2) Importing helper functions found in [`utils.js`](https://github.com/near-examples/transaction-examples/blob/master/utils.js)
+
+```js
+const nearAPI = require('near-api-js');
+const { setupNear, formatAmount } = require('./utils');
+```
+___
+
+## Setting up connection to NEAR
```

Comment:
I would also add a section on using a keypair directly to construct a transaction (vs. going to keystore to find keys)

Reply:
fixed 👍